Project.toml

```diff
@@ -0,0 +1,19 @@
+name = "ConvexHull"
+uuid = "ac7f83e5-6438-509f-b68d-98056daad062"
+authors = ["Joey Huchette"]
+version = "0.1.0"
+
+[deps]
+Combinatorics = "861a8166-3701-5b0c-9a16-15d98fcdc6aa"
+GenericLinearAlgebra = "14197337-ba66-59df-a3e3-ca00e7dcff7a"
+Polyhedra = "67491407-f73d-577b-9b50-8179a7c68029"
+Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+RowEchelon = "af85af4c-bcd5-5d23-b03a-a909639aa875"
+
```

Comment:
Add a `[compat]`  with `julia = "1"` and `Polyhedra = "0.5"`

Reply:
```
ERROR: empty intersection between Polyhedra@0.4.5 and project compatibility 0.5.0-0.5
```

Reply:
Did you do `] add Polyhedra@0.4.5` in the past ? In this case, you can do `] free Polyhedra` or do `] activate .` in the `dev/ConvexHull` folder.

Reply:
ok nevermind here, solved